src/app/components/common/node-editor/custom-node/custom-node.component.ts

```diff
@@ -165,6 +173,12 @@ export class CustomNodeComponent implements OnChanges {
     }
 
     if (!this.nodeTypes.includes(this.data?.label) && !isEmpty(this.data.controls)) {
+      if (this.elRef.nativeElement.children.length !== 0 && this.elRef.nativeElement.children[0].classList.contains('selected-node')) {
+        let boxShadowValue = this.data.label === "South" ? "0 1px 1px rgba(0, 0, 0, 0.075) inset, 0 0 8px #B6D7A8" : "0 1px 1px rgba(0, 0, 0, 0.075) inset, 0 0 8px rgb(249, 203, 156)";
+        this.elRef.nativeElement.style.boxShadow = boxShadowValue;
+      } else {
+        this.elRef.nativeElement.style.removeProperty('box-shadow');
+      }
```

Comment:
Duplicate code? line 108-113  & 176-181 are same only difference is colour code in the boxShadowValue? 
It can be moved into a separate function passing colour code as param and concatenate in the string.